### ISSUE TITLE
CLI should fail gracefully and let user know that server needs to be running

### ISSUE THREAD

**Issue by bharatagarwal** [unnamed-paas/mothership-cli#6]
![Screenshot 2019-12-10 at 11 34 10 PM](https://user-images.githubusercontent.com/15229351/70555729-a3ce9500-1ba5-11ea-89d3-291f630bcbb8.png)

**Comment by jkulton** [comment]
Yeah this shouldn't be hard. Just gracefully handle the response and let them know Mothership doesn't seem to live/isn't running at that location.
